### PR TITLE
Add Teams recommendation about WebSocket traffic

### PR DESCRIPTION
**Pull request by yirkha**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/prepare-network.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ For an in-depth discussion of network optimization, including guidance for ident
 </ul>
 <p>Each wireless vendor has its own recommendations for deploying its wireless solution. Consult your WiFi vendor for specific guidance.</p></td>
 </tr>
+<tr class="odd">
+<td>Allow WebSocket protocol</td>
+<td><p>Teams utilizes long-living bidirectional WebSocket connections to efficiently deliver chat updates, call notifications and signaling, presence status changes and other real-time notifications from cloud services to the client. If establishing WebSocket connections is not possible, a traditional HTTP long-polling transport is used as a fallback, resulting in increased bandwidth and latency. Enabling WebSocket traffic is recommended for optimal operation.</p></td>
```
Comment (JohanFreelancer9) [inline Teams/prepare-network.md:137]:
```suggestion
<td><p>Teams utilizes long-living bidirectional WebSocket connections to efficiently deliver chat updates, call notifications and signaling, presence status changes, and other real-time notifications from cloud services to the client. If establishing WebSocket connections is not possible, a traditional HTTP long-polling transport is used as a fallback, resulting in increased bandwidth and latency. We recommend that you enable WebSocket traffic for optimal operation.</p></td>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @yirkha, kindly commit the suggested changes. Many thanks.

**Comment by yirkha** [comment]
Copy review fixes committed, thanks for your help 🙂

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by yirkha** [comment]
@dstrome can you help maybe?
(I apologize for a random mention, I just picked someone more potent with closing PRs from history)
This has been open for a long time and we would like to point customers to this info in the official docs.
Thank you

**Comment by scanum** [comment]
@tiburd Hi Tina! Hope all is well. Could you please help here? Thanks!

**Pull request merged**